core/arch/arm/kernel/thread.c

```diff
@@ -89,7 +89,7 @@
 
 struct thread_ctx threads[CFG_NUM_THREADS];
 
-static struct thread_core_local thread_core_local[CFG_TEE_CORE_NB_CORE];
+struct thread_core_local thread_core_local[CFG_TEE_CORE_NB_CORE];
```

Comment:
This change should go into the "core: arm: kernel: make thread_core_local public" commit